<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RekkixMW</class>
 <widget class="QMainWindow" name="RekkixMW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rekkix, the requirements matrix !</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/rekkix.qrc">
    <normaloff>:/icon/Rekkix_logo.svg</normaloff>:/icon/Rekkix_logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_14">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mw_tabs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_configuredFiles">
       <attribute name="title">
        <string>Fichiers configurés</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QGroupBox" name="gb_configuredFiles">
          <property name="title">
           <string>Fichiers configurés</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableView" name="tv_configuredFiles"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="gb_configurationErrors">
          <property name="title">
           <string>Erreurs de configuration</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QTableView" name="tv_configurationErrors"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QProgressBar" name="progbar_analysis">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string notr="true">%p%</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_startAnalysis">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Lancer analyse</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>468</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_results">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Résultats</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <widget class="QSplitter" name="frameVerticalSplitter">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="frameFiles">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0" rowspan="2">
             <widget class="QGroupBox" name="gb_filesCoverageSummary">
              <property name="title">
               <string>Résumé de la couverture des fichiers</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QTableView" name="tv_filesCoverageSummary">
                 <property name="minimumSize">
                  <size>
                   <width>600</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::ExtendedSelection</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                  <bool>true</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="gb_upstreamDocs">
              <property name="minimumSize">
               <size>
                <width>233</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>233</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Documents en amont</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QTableView" name="tv_upstreamDocs">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::SingleSelection</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QGroupBox" name="gb_downstreamDocs">
              <property name="minimumSize">
               <size>
                <width>233</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>233</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Documents en aval</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QTableView" name="tv_downstreamDocs">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::SingleSelection</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QSplitter" name="frameHorizontalSplitter">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="gb_errors">
            <property name="title">
             <string>Erreurs</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QTableView" name="tv_errors">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QTabWidget" name="result_tabs">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_reqs">
             <attribute name="title">
              <string>Exigences définies</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QTableView" name="tv_definedReqs">
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_covering">
             <attribute name="title">
              <string>Exigences couvrantes</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QTableView" name="tv_upstreamCoverage">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_composite">
             <attribute name="title">
              <string>Exigences composites</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QTableView" name="tv_compositeReqs">
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_covered">
             <attribute name="title">
              <string>Exigences couvertes</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QTableView" name="tv_downstreamCoverage">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_generateReports">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Génération de rapport</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionOuvrir_fichier_de_configuration"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menu_Fichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionOuvrir_fichier_de_configuration">
   <property name="text">
    <string>Ouvrir fichier de configuration</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/rekkix.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>RekkixMW</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOuvrir_fichier_de_configuration</sender>
   <signal>triggered(bool)</signal>
   <receiver>RekkixMW</receiver>
   <slot>slt_loadConfigurationFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_startAnalysis</sender>
   <signal>clicked()</signal>
   <receiver>RekkixMW</receiver>
   <slot>slt_startAnalysis()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tv_filesCoverageSummary</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>RekkixMW</receiver>
   <slot>slt_fileCoverageSummary_selected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_generateReports</sender>
   <signal>clicked()</signal>
   <receiver>RekkixMW</receiver>
   <slot>slt_generateReports()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slt_loadConfigurationFile()</slot>
  <slot>slt_startAnalysis()</slot>
  <slot>slt_fileCoverageSummary_selected(QModelIndex)</slot>
  <slot>slt_generateReports()</slot>
 </slots>
</ui>
